server/data/model/clustersSummaryResponse.ts

```diff
@@ -0,0 +1,16 @@
+import { PaginationDetails } from './paginationDetails'
+
+export interface SourceSystemComposition {
+  sourceSystem: string
+  count: number
+}
+
+export interface ClusterSummary {
+  uuid: string
+  recordComposition: SourceSystemComposition[]
+}
+
+export interface ClustersSummaryResponse {
```

Comment:
this (and the filename) looks like it should be named ClusterSummaryResponse to match the above ClusterSummary

Reply:
or rename all to ClustersSummary (prefer the above though) but in any case consistency will be easier to deal with